Installation/include/CGAL/config.h

```diff
@@ -422,6 +422,22 @@ using std::max;
 #  define CGAL_DEPRECATED
 #endif
 
+// Macro to trigger deprecation warnings with a custom message
+#ifdef CGAL_NO_DEPRECATION_WARNINGS
+#  define CGAL_DEPRECATED_MSG
```

Comment:
There is an error, here. That lines defines a macro without argument. That will lead to a compilation error for any use of that macro with an argument.